docs/browser-support.mdx

```diff
@@ -0,0 +1,14 @@
+---
+name: Browser support
+route: /browser-support
+cardHeadline: Browser support
+cardSubHeadline: Creating a consistent experience across browsers and devices.
+---
+
+## Supported Devices
+
+The design system should be universally accessible, which means that the components should work in the browsers and devices used to access
+the project, which is using the design system. Components might not look perfect in all scenarios, but everything should still be
```

Comment:
Can we re-phrase this a bit? 
`which means that the components should work in the browsers and devices used to access
the project, which is using the design system` or it is just my English? haha

Perhaps, `The design system should be universally accessible. Components might not look perfect in all scenarios, but everything should still be accessible to the user` _(so we just drop that sentence)_

Reply:
That's fine with me, I've updated the documentation!